components/FeatureHighlight/BUILD

```diff
@@ -88,6 +90,39 @@ mdc_objc_library(
     visibility = ["//visibility:private"],
 )
 
+mdc_examples_objc_library(
+    name = "ObjcExamples",
+    deps = [
+        ":ColorThemer",
+        ":TypographyThemer",
+        ":FeatureHighlight",
```

Comment:
Nit: Order of dependencies (here and below). Please consider running `buildifier` on your files.